upgrade_tests/helpers/existence.py

```diff
@@ -89,15 +90,14 @@
  )
 
 # Attributes where 'name' as key to fetch component property data
-attribute_keys = dict.fromkeys(
+attribute_keys.update(dict.fromkeys(
```

Comment:
Fix:
We are appending the dictionary of entities(by unique name) with dictonary of entities(by unique id).